hardhat.config.ts

```diff
@@ -103,10 +103,13 @@ const config: HardhatUserConfig = {
       timeout: 12000000,
     },
     bscmainnet: {
-      url: "http://127.0.0.1:1248",
+      url: "https://bsc-dataseed.binance.org",
```

Comment:
I wouldn't do this change. We should use Frame for the deployment of contracts to mainnet